packages/compartment-mapper/test/test-optional.js

```diff
@@ -31,15 +31,13 @@ scaffold(
       ['one', 'two'],
       'expected errors',
     );
-    t.true(
-      result.errors.one.message.startsWith(
-        `Cannot find external module "missing-one" in package`,
-      ),
+    t.regex(
+      result.errors.one.message,
+      /^Cannot find external module "missing-one" in package.*/,
```

Comment:
No behavioral change in this test, I just made failure messages more helpful (while these tests were not yet passing)